pkgs/by-name/ap/appmenu-glib-translator/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  fetchFromGitLab,
+  glib,
+  gtk3,
+  meson,
+  ninja,
+  pkg-config,
+  gobject-introspection,
+  vala,
+  stdenv,
+  wrapGAppsHook3,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "appmenu-glib-translator";
+  version = "25.04";
+
+  src = fetchFromGitLab {
+    owner = "vala-panel-project";
+    repo = "vala-panel-appmenu";
+    tag = finalAttrs.version;
+    fetchSubmodules = true;
+    hash = "sha256-v5J3nwViNiSKRPdJr+lhNUdKaPG82fShPDlnmix5tlY=";
+  };
+
+  sourceRoot = "source/subprojects/appmenu-glib-translator";
+
+  nativeBuildInputs = [
+    meson
+    ninja
+
+    pkg-config
+    wrapGAppsHook3
+    vala
+  ];
+
+  buildInputs = [
```

Comment:
Both GLib and gdk-pixbuf are in Requires field of the pkg-config file so they should go to `propagatedBuildInputs`.

Though gdk-pixbuf is only a private dependency so the pkg-config file should be fixed upstream, and then it can go back to `buildInputs`.

**Edit:** Actually, the code depending on gdk-pixbuf is [commented](https://gitlab.com/vala-panel-project/vala-panel-appmenu/-/commit/919686a7d75c26629db6aa75b0f305b7c5e4556a)/[preprocessed](https://gitlab.com/vala-panel-project/vala-panel-appmenu/-/commit/8c0880e7b2ad9eba3681005824a9713f5833f192) out so the `gdk-pixbuf` dependency can be removed to no ill effect.